### DISCUSSION TITLE
Where can I obtain rabbit_ Password_ Hashing_ Sha256 algorithm,How to implement Golang

### DISCUSSION THREAD

**Discussion by charlienss** [rabbitmq/rabbitmq-server#9029] [Other] [answered]
Where can I obtain rabbit_ Password_ Hashing_ Sha256 algorithm,How to implement Golang

**Comment by michaelklishin** [comment, marked as answer]
It's been implemented in Go a number of years ago, [right here](https://github.com/michaelklishin/rabbit-hole/blob/main/users.go#L194-L230) :)

**Reply by charlienss** [reply]
Thank you very much
